### ISSUE TITLE
purple stripes on side of screen

### ISSUE THREAD

**Issue by Domitilla11** [Lyall/WukongTweak#9]
hey, i really like the tweaks and am very happy with the results, one thing though i encountered while playing. In cutscenes an at a shrine  purple stripes/lines appear on the side of the screen. Maybe for info. i play on a 2k monitor with 2k resolution in game, borderless and 16:9. The stripes are kinda annoying, if this would be fixed, the tweaks are a 10/10.

**Comment by Lyall** [comment]
Can you please post your `WukongTweak.log` file? Also can you please post a screenshot of the issue if you can?

**Comment by Domitilla11** [comment]
> Can you please post your `WukongTweak.log` file? Also can you please post a screenshot of the issue if you can?

yes of course, i did a screenshot in the cutscene, tried also to do one at shrine but needed to be quick, so not as much on that screenshot. when i am in game, i mean going around in the world, there is nothing of that purple stripe.
![b1   25 08 2024 15_41_21](https://github.com/user-attachments/assets/fc5e7771-352e-4a98-85c6-89349518aad8)
![b1   25 08 2024 15_26_54](https://github.com/user-attachments/assets/47cadcc5-b34c-4274-a46f-eb455afc38be)
![purple stripes on side of screen · Issue #9 · Lyall_WukongTweak - Google Chrome 25 08 2024 15_41_11](https://github.com/user-attachments/assets/fa472421-cca9-4c38-8b62-48b3c17e5949)
![WukongTweak log – Notepad 25 08 2024 15_46_11](https://github.com/user-attachments/assets/731e9da9-18d7-4209-8988-1a31402bacd6)

**Comment by Lyall** [comment]
Does this same visual error occur without WukongTweak installed?

**Comment by Domitilla11** [comment]
> Does this same visual error occur without WukongTweak installed?

No, only appeared since i installed the tweak. I also deinstalled it and then it disappeared. Hope this helps

**Comment by Lyall** [comment]
> > Does this same visual error occur without WukongTweak installed?
> 
> No, only appeared since i installed the tweak. I also deinstalled it and then it disappeared. Hope this helps

Can you try this test build and see if it fixes it?

[WukongTweak_v0.8.3.zip](https://github.com/user-attachments/files/16743085/WukongTweak_v0.8.3.zip)

**Comment by Domitilla11** [comment]
unfortunately didn't help

**Comment by Lyall** [comment]
Hmm,. that's a really weird one then. I can't think of what would be causing it. Perhaps forcing the game to recompile shaders may help? 

It's located at `%LOCALAPPDATA%\b1\Saved`, just delete the large .USHADERPRECACHE file then relaunch the game. 

If that doesn't work for you then I'd recommend trying to disable parts of WukongTweak in the ini until it makes a difference.
